services/server/server.go

```diff
@@ -247,13 +270,13 @@ func (s *Server) CheckUpdates(ctx context.Context, req *serverpb.CheckUpdatesReq
 
 	if req.Force {
 		if err := s.supervisord.ForceCheckUpdates(ctx); err != nil {
-			return nil, err
+			return s.noInternetResponse(ctx)
```

Comment:
We shouldn't ignore error message.
Can we return error message and installed version together?
If we can't we should at least log the error and maybe add new flag to return only installed version.

Reply:
I can modify struct to have field for error where we can mention that PMM is offline, but this changes also needs some changes on FE.

About flag is it really needed? Like if we will modify FE then we can do it on FE. What do you think?

Reply:
Flag looks more proper because it's consistent with other out APIs.